src/app/shared/helper-services/update-data.service.ts

```diff
@@ -91,6 +95,14 @@ export class UpdateDataService {
         return fsat;
     }
 
+    addFsatOperations(fsat: FSAT): FSAT {
+        if(!fsat.fsatOperations) {
+            fsat.fsatOperations.cost = 0.06;
+            fsat.fsatOperations.operatingHours = 8760;
+        }
```

Comment:
I realized we should probably check if the value can come from the field data that they had set.

```
if(fsat.fieldData['operatingHours']){
fsat.fsatOperations.operatingHours = fsat.fieldData['operatingHours]
}else{
fsat.fsatOperations.operatingHours = 8760
}
```

Reply:
Then it won't override existing changes to assessments

Reply:
Do the same thing for cost..